example/src/lib.rs

```diff
@@ -11,54 +11,73 @@ mod ffi {
     use writeable::Writeable;
 
     #[diplomat::opaque]
+    /// A decimal number. See [the Rust docs](https://unicode-org.github.io/icu4x-docs/doc/fixed_decimal/decimal/struct.FixedDecimal.html) for more information.
     pub struct ICU4XFixedDecimal(pub FixedDecimal);
 
     impl ICU4XFixedDecimal {
+        /// Construct an [`ICU4XFixedDecimal`] from an integer.
         fn new(v: i32) -> Box<ICU4XFixedDecimal> {
             Box::new(ICU4XFixedDecimal(FixedDecimal::from(v)))
         }
 
+        /// Multiply the [`ICU4XFixedDecimal`] by a given power of ten.
+        /// See [the Rust docs](https://unicode-org.github.io/icu4x-docs/doc/fixed_decimal/decimal/struct.FixedDecimal.html#method.multiply_pow10) for more information.
         fn multiply_pow10(&mut self, power: i16) {
             self.0.multiply_pow10(power).unwrap();
         }
 
+        /// Invert the sign of the [`ICU4XFixedDecimal`].
+        /// See [the Rust docs](https://unicode-org.github.io/icu4x-docs/doc/fixed_decimal/decimal/struct.FixedDecimal.html#method.negate) for more information.
```

Comment:
Also, the URL prefix for the ICU4X docs should be an argument to the tool, certainly not hard-coded in the source.

Reply:
+1

Reply:
Well, I'd say we should wait for intra-doc-links to become possible and _then_ you can specify the root in an argument.

You'd basically say `[the Rust docs](struct fixed_decimal::decimal::FixedDecimal)` or something

Reply:
I assume this is covered by #21?